app/uk/gov/hmrc/customs/declarations/metrics/controllers/CustomsDeclarationsMetricsController.scala

```diff
@@ -17,10 +17,8 @@
 package uk.gov.hmrc.customs.declarations.metrics.controllers
 
 import javax.inject.{Inject, Singleton}
-
 import play.api.i18n.{I18nSupport, Messages, MessagesApi}
 import play.api.libs.json.{JsError, JsSuccess, JsValue, Json}
-import play.api.mvc
 import play.api.mvc.{Action, BodyParser}
 import uk.gov.hmrc.customs.api.common.controllers.ErrorResponse.errorBadRequest
 import uk.gov.hmrc.customs.api.common.controllers.{ErrorResponse, ResponseContents}
```

Comment:
protected vals like `protected val nonJsonBodyErrorMessage = "Request does not contain a valid JSON body"
` in controller should be private if possible. Also applies to `protected def invalidJsonErrorResponse(jsError: JsError)(implicit messages: Messages, hc: HeaderCarrier): ErrorResponse = {`.

Also applies to:
- `HeaderValidator`